content/metastore-integrations/docs/hive-metastore.md

```diff
@@ -151,4 +157,213 @@ aws emr-containers start-job-run --cli-input-json file:///Spark-Python-in-s3-hms
+
+The CDK/CFN template is available to simplify the installation against a new EKS cluster. If you have an existing EKS cluster, the prerequisite details can be found in the [github repository](https://github.com/melodyyangaws/hive-emr-on-eks#41-run-the-thrift-service-as-a-sidecar-in-spark-drivers-pod)
+
+**sidecar_hivethrift_eks.py:**
+
+```python
+import sys
+from pyspark.sql import SparkSession
+
+spark = SparkSession \
+    .builder \
+    .config("spark.sql.warehouse.dir",environ['warehouse_location']) \
+    .enableHiveSupport() \
+    .getOrCreate()
+
+spark.sql("SHOW DATABASES").show()
+spark.sql("CREATE DATABASE IF NOT EXISTS `demo`")
+spark.sql("DROP TABLE IF EXISTS demo.amazonreview4")
+spark.sql("CREATE EXTERNAL TABLE `demo`.`amazonreview4`( `marketplace` string,`customer_id`string,`review_id` string,`product_id` string,`product_parent` string,`product_title` string,`star_rating` integer,`helpful_votes` integer,`total_votes` integer,`vine` string,`verified_purchase` string,`review_headline` string,`review_body` string,`review_date` date,`year` integer) STORED AS PARQUET LOCATION '"+sys.argv[1]+"/app_code/data/toy/'")
+
+spark.stop()
+```
+
+**Request:**
+
+Now that the HMS is running inside your Spark driver, it share common attributes such as the network config, the `spark.hive.metastore.uris` can set to "thrift://localhost:9083". Don't forget to assign the sidecar pod template to the Spark Driver like this `"spark.kubernetes.driver.podTemplateFile": "s3://'$S3BUCKET'/app_code/job/sidecar_hms_pod_template.yaml"` 
+
+For more details, check out the [github repo](https://github.com/melodyyangaws/hive-emr-on-eks#41-run-the-thrift-service-as-a-sidecar-in-spark-drivers-pod)
+
+```bash
+#!/bin/bash
+# test HMS sidecar on EKS
+aws emr-containers start-job-run \
+--virtual-cluster-id $VIRTUAL_CLUSTER_ID \
+--name sidecar-hms \
+--execution-role-arn $EMR_ROLE_ARN \
+--release-label emr-6.3.0-latest \
+--job-driver '{
+  "sparkSubmitJobDriver": {
+      "entryPoint": "s3://'$S3BUCKET'/app_code/job/sidecar_hivethrift_eks.py",
+      "entryPointArguments":["s3://'$S3BUCKET'"],
+      "sparkSubmitParameters": "--conf spark.driver.cores=1 --conf spark.executor.memory=4G --conf spark.driver.memory=1G --conf spark.executor.cores=2"}}' \
+--configuration-overrides '{
+    "applicationConfiguration": [
+      {
+        "classification": "spark-defaults", 
+        "properties": {
+          "spark.kubernetes.driver.podTemplateFile": "s3://'$S3BUCKET'/app_code/job/sidecar_hms_pod_template.yaml",
+          "spark.hive.metastore.uris": "thrift://localhost:9083"
+        }
+      }
+    ], 
+    "monitoringConfiguration": {
+      "s3MonitoringConfiguration": {"logUri": "s3://'$S3BUCKET'/elasticmapreduce/emr-containers"}}}'
+```
+
+### **5-Hudi + Remote Hive metastore integration**
```

Comment:
Is this a repeat of info from here?
content/metastore-integrations/docs/aws-glue.md

Reply:
no, this is Hudi sync with remote RDS hive metastore, the other one is sync with Glue catalog.